docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -420,13 +567,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Classification Model                                 | Accuracy Score\n",
+      "SVC using kernel as a callable function              |       1.00\n",
+      "SVC using precomputed kernel matrix                  |       1.00\n",
+      "QSVC                                                 |       1.00\n",
+      "-------------------------------------------------------------------------\n",
+      "Clustering Model                                     | Accuracy Score\n",
+      "Spectral Clustering using precomputed kernel matrix  |       0.73\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"Classification Model                                 | Accuracy Score\")\n",
+    "print(\n",
+    "    f\"SVC using kernel as a callable function              | {adhoc_score_callable_function:10.2f}\"\n",
+    ")\n",
+    "print(\n",
+    "    f\"SVC using precomputed kernel matrix                  | {adhoc_score_precomputed_kernel:10.2f}\"\n",
+    ")\n",
+    "print(f\"QSVC                                                 | {qsvc_score:10.2f}\")\n",
+    "print(f\"-------------------------------------------------------------------------\")\n",
+    "print(f\"Clustering Model                                     | Accuracy Score\")\n",
+    "print(f\"Spectral Clustering using precomputed kernel matrix  | {cluster_score:10.2f}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What was learned\n",
```

Comment:
> What was learned

Should it be a numbered section?

Reply:
Well, @adekusar-drl, the initial idea was to have `Overview`, `Introduction` and this last section as non-numbered. I have been thinking though about alternative names like `Summary`... what do you think?

Reply:
`Lessons learned`, `Conclusion` are good, I think.